types/subleveldown/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for subleveldown 4.1
+// Project: https://github.com/level/subleveldown
+// Definitions by: Carson Farmer <https://github.com/carsonfarmer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { LevelUp } from 'levelup';
+import { ErrorValueCallback, ErrorCallback, AbstractGetOptions } from 'abstract-leveldown';
+
+declare namespace sub {
+    interface Options {
+        separator?: string;
+        open?: ((callback?: ErrorCallback) => any);
```

Comment:
Please Use `void` as [return type of callback][callback].

[callback]: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#return-types-of-callbacks

Reply:
Done